scripts/build_exe_win7-amd64.py

```diff
@@ -23,8 +23,8 @@
 
 # -------------------------------------------------------------------------------
 #
-# CHIPSEC: Platform Hardware Security Assessment Framework
-# (c) 2010-2012 Intel Corporation
+# CHIPSEC: Platform Security Assessment Framework
+# (c) 2010-2020 Intel Corporation

```

Comment:
Only Intel employees should be updating the copyright year for Intel.  You can either add your own/company copyright or waive the rights to add to the copyright.